### ISSUE TITLE
join error in get_meds_files

### ISSUE THREAD

**Issue by vargatn** [esheldon/ngmixer#74]
In `get_meds_files` the variable `dir` shadows the built in dir() function, and is otherwise not defined (although seems to be defined in other functions in that module). This results in a TypeError when attempting an os.path.join.

**Comment by esheldon** [comment]
Yes, I'm aware of it (niall also reported), but somehow the installer is fixing this bug, perhaps due to the 2to3 conversion.

In any case I will fix it.
